processor/redactionprocessor/go.mod

```diff
@@ -0,0 +1,33 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/processor/redactionprocessor
+
+go 1.17
+
+require (
+	github.com/stretchr/testify v1.7.0
+	go.opentelemetry.io/collector v0.39.0
+	go.opentelemetry.io/collector/model v0.39.0
```

Comment:
This needs to be in sync with what we have in the main branch at the moment of the merge. This is currently `v0.39.1-0.20211117203239-e23c9d0a0183`

Reply:
@jpkrohling Updated to `v0.39.1-0.20211117203239-e23c9d0a0183`

Reply:
Sorry, looks like this got updated once again to v0.39.1-0.20211119172502-53d057f9c0e7 by #6390.

Reply:
@jpkrohling Updated to `v0.39.1-0.20211119172502-53d057f9c0e7`